### PR TITLE
Adding support to services publication

### LINKED ISSUE #93: Add services support to the CLI

### PR DESCRIPTION
**Pull request by aaitor**
## Description

Add a description of your changes here.

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] Documentation

### CODE REVIEW

File: resources/commands.json [hunk 1]
```diff
@@ -282,6 +307,94 @@
...
+						"default": "721",
+						"hidden": true,
+						"description": "The NFT type"
+					}					
+				]
+			}, {
+				"name": "register-service",
+				"description": "Register a new web service",
+				"details": "This command registers a new web service on the Nevermined network. The service can be the representation of any regular Web Service (REST, JSON-RPC, etc).",
+				"examples": ["ncli assets register-service --name 'My AI Bot' --author 'John Doe' --price 0 --language python --entrypoint 'python word_count.py' --container python:3.8-alpine --urls https://www.apache.org/licenses/LICENSE-2.0 --contentType text/plain"],
```
Comment (r-marques) [inline resources/commands.json]: maybe this should be a better example

File: resources/commands.json [hunk 2]
```diff
@@ -282,6 +307,94 @@
...
+					{
+						"name": "dateCreated",
+						"type": "string",
+						"description": "When the asset was created"
+					},
+					{
+						"name": "access",
+						"type": "string",						
+						"default": "direct",
+						"description": "The type of access control associated to the asset. If `direct` the asset will allow direct payment, if `susbcription` it will allow access via NFT susbcription, if both it will configure both methods."
```
Comment (r-marques) [inline resources/commands.json:353]: do web services work without an nft subscription?

File: resources/networks.json [hunk 3]
```diff
@@ -121,7 +121,7 @@
 			"web3ProviderUri": "http://contracts.nevermined.localnet",
 			"marketplaceUri": "http://marketplace.nevermined.localnet",
 			"graphHttpUri": "http://localhost:9000/subgraphs/name/nevermined-io/development",
-			"neverminedNodeUri": "http://node.nevermined.localnet",
+			"neverminedNodeUri": "http://localhost:8030",
```
Comment (r-marques) [inline resources/networks.json]: this will probably need to be changed back

File: src/commands/nfts/getJWT.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  const { did } = argv
+
+  logger.info(
+    chalk.dim(`Gets JWT Access Token ${chalk.whiteBright(did)}`)
+  )
+
+  let ddo
+  
+  try {
+    ddo = await nvm.assets.resolve(did)
```
Comment (r-marques) [inline src/commands/nfts/getJWT.ts:24]: is there a need to resolve the did?

File: resources/commands.json (hunk 2 above)
Comment (aaitor) [reply resources/commands.json:353]: fixed to get a default option that is hidden

File: resources/networks.json (hunk 3 above)
Comment (aaitor) [reply resources/networks.json]: :+1:

File: resources/commands.json (hunk 1 above)
Comment (aaitor) [reply resources/commands.json]: :+1:

File: src/commands/nfts/getJWT.ts (hunk 4 above)
Comment (aaitor) [reply src/commands/nfts/getJWT.ts:24]: it's just for failing if the DID doesn't exist

**Review by r-marques — APPROVED** [review]

**Pull request merged**